pxr/usdImaging/usdImagingGL/engine.cpp

```diff
@@ -1153,7 +1153,13 @@ UsdImagingGLEngine::SetRendererSetting(TfToken const& id, VtValue const& value)
     }
 
     TF_VERIFY(_renderDelegate);
-    _renderDelegate->SetRenderSetting(id, value);
+    // usdviewq passes a double as VtValue since Python has no 'float' type
+    if (value.IsHolding<double>()) {
+        _renderDelegate->SetRenderSetting(
+            id, VtValue(static_cast<float>(value.UncheckedGet<double>())));
+    } else {
```

Comment:
I'd also make the conversion chain just about converting `value` or issuing an error and returning, so that there's only one call to SetRenderSetting() at the end.